src/components/Filters/Filters.tsx

```diff
@@ -78,6 +72,7 @@ const Filters: React.FC<FiltersProps> = ({
   const [filtersValues, setFiltersValues] = useState<Array<Filter>>(null);
   const [isDefaultState, setIsDefaultState] = useState(true);
   const [hasUnappliedFilters, setHasUnappliedFilters] = useState(false);
+  const [validsValues, setValidsValues] = useState<boolean[]>([true]);
```

Comment:
Isn't `validValues` correct name?

Reply:
Shouldn't be the default value an empty array? Or it should be derived from `state`

Reply:
the default is an array of one true element because it's one Field / FilterRow. We can move it to the first useEffect but it's the same

Reply:
If `state` passed down to Filters will include 5 "filters" definitions then it should include five `true`s. Therefore it should be derived from state.length